upf/conditional_effects_remover.py

```diff
@@ -46,43 +53,29 @@ def get_rewritten_problem(self) -> Problem:
         if self._unconditional_problem is not None:
             return self._unconditional_problem
         #cycle over all the actions
-        #Note that a different environment might be needed when multy-threading
-        action_stack: List[Tuple[str, Action]] = []
-        new_problem = self._create_problem_copy(action_stack)
+        #NOTE that a different environment might be needed when multy-threading
+        action_stack: List[Action] = []
+        new_problem, action_stack = self._create_problem_copy()
 
         while len(action_stack) > 0:
```

Comment:
Why not a `for` on the list of conditional actions?

Reply:
Right now we don't have a method that gives the conditional/unconditional actions of a problem.

I will resolve this asap